src/components/CertsComponent.vue

```diff
@@ -19,21 +24,28 @@
             type: String,
             default: '',
         },
-        typeOfDoc: {
```

Comment:
is it path or link to place in db?

Reply:
place in DB

Reply:
![image](https://user-images.githubusercontent.com/25306811/211864067-e2f6e5fb-2b31-45d1-9c8b-d952ef9a5864.png)
#